src/common/components/TableResponsive.js

```diff
@@ -259,8 +261,12 @@ const SearchBar = props => {
         sx={{ flexGrow: 2, width: { xs: '100%' } }}
         InputProps={{
           sx: {
+            borderColor: theme.palette.gray.dark2,
             marginBottom: 2,
             padding: 0,
+            '& ::placeholder': {
+              opacity: 0.65,
+            },
           },
```

Comment:
Suggestion: You can get the `theme` from inside the sx property without importing it
```suggestion
          sx: theme => ({
            borderColor: theme.palette.gray.dark2,
            marginBottom: 2,
            padding: 0,
            '& ::placeholder': {
              opacity: 0.65,
            },
          }),
```